www/artworks/new.php

```diff
@@ -154,14 +166,7 @@
 						/>
 					</label>
 				</fieldset>
-				<label>
-					Link to museum page
-					<input
-						type="url"
-						name="pd-proof-museum-link"
-						value="<?= Formatter::ToPlainText($artwork->MuseumPage) ?>"
-					/>
-				</label>
+				<p>See the <a href="/manual/latest/10-art-and-images#10.3.3.7">Manual of Style</a> for full details on US-PD clearance.</p>
```

Comment:
Consider:

```html
<p>See the <a href="/manual/latest/10-art-and-images#10.3.3.7">US-PD clearance section of the <abbr class="acronym">SEMoS</abbr></a> for details.</p>
```

to match the link style in the step-by-step guide, e.g.,

https://github.com/standardebooks/web/blob/c014b8edbd2318e9eb271afb8b05c55386ae3a3d/www/contribute/producing-an-ebook-step-by-step.php#L618